org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -3717,3 +3740,93 @@ class ColorPresentation {
 		this.additionalTextEdits = additionalTextEdits
 	}
 }
+
+/**
+ * Parameters for the semantic highlighting (server-side) push notification.
+ *
+ * @experimental
+ */
+@JsonRpcData
```

Comment:
We have used the `com.google.common.annotations.Beta` annotation for other proposed extensions.